cpp/oneapi/dal/data/graph_service_functions.hpp

```diff
@@ -15,6 +15,9 @@
  * limitations under the License.
  *******************************************************************************/
 
+/// @file
+/// Contains the service functionality for the graph objects
```

Comment:
Contains the service **function** for the graph objects

_Better this way, but if you are sure, that it distorts the meaning, feel free to keep it without changes_